File: src/main/java/com/matag/admin/game/findactive/ActiveGameResponse.java
package com.matag.admin.game.findactive;

import java.time.LocalDateTime;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Value;

@Value
@AllArgsConstructor
@JsonDeserialize(builder = ActiveGameResponse.ActiveGameResponseBuilder.class)
@Builder
public class ActiveGameResponse {
  Long gameId;
  LocalDateTime createdAt;
  String playerName;
  String playerOptions;
  String opponentName;
  String opponentOptions;

  @JsonPOJOBuilder(withPrefix = "")
  public static class ActiveGameResponseBuilder {

  }
}

File: src/main/java/com/matag/admin/game/findactive/ActiveGameResponse.kt
package com.matag.admin.game.findactive;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ActiveGameResponse {
  Long gameId;
  LocalDateTime createdAt;
  String playerName;
  String playerOptions;
  String opponentName;
  String opponentOptions;
}
